flow-nodes/api-builder-plugin-petstore/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "api-builder-plugin-petstore",
+	"version": "1.0.0",
+	"description": "A flow-node to connect to the petstore API",
+	"author": "",
+	"license": "ISC",
+	"keywords": [
+		"amplify",
+		"api-builder"
+	],
+	"engines": {
+		"node": ">= 12.13"
+	},
+	"main": "index.js",
+	"files": [
+		"index.js",
+		"openapi",
+		"README.md"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/axway/api-builder-examples"
+	},
+	"dependencies": {
+		"@axway/api-builder-oas-flow-node": "^1.0.0"
+	},
+	"peerDependencies": {
+		"@axway/api-builder-runtime": "^4.5.0"
+	},
+	"devDependencies": {
+		"@axway/api-builder-sdk": "^0.2.0",
+		"chai": "^4.1.2",
+		"mocha": "^6.1.4"
+	},
+	"scripts": {
+		"test": "mocha ./test --recursive -R spec"
```

Comment:
Still manage to find test.js but it is coincidence. There is no 'test' directory

Reply:
./test will match the directory as well as the file, but you're right it may not be expected. although we can use this as a lucky coincidence as it will still work if the user wants to change where the tests live

Reply:
agreed - make ./test folder